README.md

```diff
@@ -7,8 +7,7 @@ developed by Google but not by the core Flutter team.
 
 ## Issues
 
-Please file any issues, bugs, or feature requests in the [this
-repo](https://github.com/google/flutter.widgets/issues/new).
+Please file any issues, bugs, or feature requests [here](https://github.com/google/flutter.widgets/issues/new).
```

Comment:
I think we should prefer descriptive links, especially since they're better for screen-readers.

In this case, perhaps just:

+ Please [file any issues, bugs, or feature requests](https://github.com/google/flutter.widgets/issues/new).